src/Microsoft.DotNet.Build.Tasks/Microsoft.DotNet.Build.Tasks.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If VS changes the encoding let's keep the encoding that VS wants.  Otherwise we'll always be reverting this line when we change the file.